web_crawled_data_retriever/tests/web_crawling_test.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+"""
+Web Crawling Test Script
+
+This script tests the functionality of the Web Crawling Data Retriever kit using unittest.
+
+Usage:
+    python tests/web_crawling_test.py
+
+Returns:
+    0 if all tests pass, or a positive integer representing the number of failed tests.
+"""
+
+import os
+import sys
+import yaml
+import unittest
+import logging
+import time
+
+# Setup logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+# Setup paths and global variables
+current_dir = os.getcwd()
+kit_dir = current_dir
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
```

Comment:
I think would be better to have the absolute path of files instead of depending that this is executed from the repo dir 

current_dir = os.path.dirname(os.path.abspath(__file__))